tests/api2/test_003_network.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python3
+
+# Author: Eric Turgeon
+# License: BSD
+
+import pytest
+import sys
+import os
+
+apifolder = os.getcwd()
+sys.path.append(apifolder)
+from auto_config import ha, user, password, ip
+from functions import GET, PUT, SSH_TEST
+from config import *
+
+if ha and "domain" in os.environ:
+    domain = os.environ["domain"]
+    gateway = os.environ["gateway"]
+    hostname = os.environ["hostname"]
+    hostname_b = os.environ["hostname_b"]
```

Comment:
We need to configure `hostname_virtual` on HA systems.

Reply:
Ah, it was not in the original script.

Reply:
added it